components/view-components/VideoComponent.vue

```diff
@@ -111,42 +106,15 @@ const props = defineProps({
 <style lang="postcss" scoped>
 .video-component {
   &__layout-flex {
-    display: grid @(--md) flex;
```

Comment:
All padding is already handled by the columns and when the column is direct child of a row everything works as intended out of the box. :-)